taxjar/taxjar-api-docs
# Updates Node

Updates Node version in the Dockerfile to be consistent with taxjar-developers repo version. 

To test:
```
docker-compose build
docker-compose up -d
```
Visit http://localhost:4567/reference/ or http://localhost:4567/guide/
Docker should build and start successfully and you should be able to navigate to the pages.
